WebRoot/css/zm.css

```diff
@@ -6529,7 +6487,7 @@ a.ZmLinkDisabled {
 }
 
 .ZmMailAttachmentIcon,
-.ZmMailAttachmentIcon .svg-icon {
+.ZmMailAttachmentIcon svg.svg-icon {
```

Comment:
minor, should this go to skin.css as all svg icon related properties are moved there?